### PR TITLE
virtal column stored generated fix

### PR DESCRIPTION
**Pull request by crspecter**
fix the column define stored generated, go-mysql can identify

close https://github.com/go-mysql-org/go-mysql/issues/625

### CODE REVIEW

File: schema/schema.go [hunk 1]
```diff
@@ -145,7 +145,7 @@ func (ta *Table) AddColumn(name string, columnType string, collation string, ext
 
 	if extra == "auto_increment" {
 		ta.Columns[index].IsAuto = true
-	} else if extra == "VIRTUAL GENERATED" {
+	} else if extra == "VIRTUAL GENERATED" || extra == "STORED GENERATED" {
 		ta.Columns[index].IsVirtual = true
```
Comment (lance6716) [inline schema/schema.go:149]: the name is `IsVirtual`, so don't save true for "STORED GENERATED"

Comment (crspecter) [reply schema/schema.go:149]: They all belong to automatically generated columns. Currently, go-mysql does not save the extra infomation. Need to define another variable in column to save it.

**Review by crspecter — COMMENTED** [review]
solve issue #625

File: go.sum [hunk 2]
```diff
@@ -1,5 +1,7 @@
 github.com/BurntSushi/toml v0.3.1 h1:WXkYYl6Yr3qBf1K79EBnL4mak0OimBfB0XUf9Vl28OQ=
 github.com/BurntSushi/toml v0.3.1/go.mod h1:xHWCNGjB5oqiDr8zfno3MHue2Ht5sIBksp03qcyfWMU=
+github.com/crspecter/go-mysql v1.3.2 h1:/aOj3ptsZu6+gDpazMkR6jlSJGNa+LSs8Q1Y4IIvfgk=
```
Comment (lance6716) [inline go.sum:3]: you can run `go mod tidy` to remove these unnecessary dependencies.

**Review by lance6716 — COMMENTED** [review]
rest lgtm

**Review by crspecter — COMMENTED** [review]
remove unnecessary dependencies

**Comment by crspecter** [comment]
remove unnecessary dependencies

**Comment by lance6716** [comment]
> remove unnecessary dependencies

please use `go mod tidy` to remove all unnecessary dependencies

File: schema/schema.go (hunk 1 above)
Comment (skoef) [reply schema/schema.go:150]: Wouldn't `IsStored` for `STORED GENERATED` be more consistent with `IsVirtual` for `VIRTUAL GENERATED`?

**Pull request merged**